diw.planx-pla.net/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "notes": [
+    "This is the dev environment manifest",
+    "That's all I have to say"
+  ],
+  "jenkins": {
+    "autodeploy": "yes"
+  },
+  "versions": {
+    "arborist": "quay.io/cdis/arborist:master",
+    "arranger": "quay.io/cdis/arranger:master",
```

Comment:
you can remove arranger and the arranger block. we are not using this service in gen3